src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --font-family: Roboto, sans-serif;
+  --color: #00acdc;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --white: #fff;
+  --blue-accent: #00acdc;
+  --gray-elements: #f3f3f3;
+  --yellow-stars: #ffde6a;
+}
```

Comment:
use sass vars

Reply:
що саме тут не так????
прописано все за для виконання в коді в подальшому, в чому помилка?

Reply:
Hi, you need to use SCSS variables instead of CSS variables 
<img width="891" alt="image" src="https://github.com/user-attachments/assets/4171f0e1-99e0-484e-9140-3c02ddd36c51">